File: iosApp/iosApp/home/model/HomeUiState.swift
import Foundation
import shared

struct HomeUiState: Equatable {
    var profileImageUrl: String
    var items: [Dday]
    
    // 기본 상태를 제공하는 static 속성
    static let `default` = HomeUiState(
        profileImageUrl: "",
        items: []
    )
}

File: shared/src/commonMain/kotlin/com/jooheon/dday/domain/model/HomeUiState.kt
package com.jooheon.dday.domain.model

data class HomeUiState(
    val profileImageUrl: String,
    val items: List<Dday>,
) {
    companion object {
        val default = HomeUiState(
            profileImageUrl = "",
            items = listOf(
                Dday.default, Dday.default, Dday.default,
            ),
        )
    }
}